codalab/lib/bundle_cli.py

```diff
@@ -497,26 +499,39 @@ def simple_group_str(info):
     @staticmethod
     def parse_target(client, worksheet_uuid, target_spec):
         """
-        Helper: A target_spec is a bundle_spec[/subpath].
+        Helper: A target_spec is a [worksheet_spec//]bundle_spec[/subpath].
         """
+        worksheet_spec = None
+        if '//' in target_spec:
+            worksheet_spec, target_spec = target_spec.split('//', 1)
         if os.sep in target_spec:
             bundle_spec, subpath = tuple(target_spec.split(os.sep, 1))
         else:
             bundle_spec, subpath = target_spec, ''
+
+        if worksheet_spec:
+            worksheet_uuid = BundleCLI.resolve_worksheet_uuid(client, '', worksheet_spec)
+
         # Resolve the bundle_spec to a particular bundle_uuid.
         bundle_uuid = BundleCLI.resolve_bundle_uuid(client, worksheet_uuid, bundle_spec)
         return (bundle_uuid, subpath)
 
     def parse_target_specs(self, items):
         targets = []
         for item in items:
-            if ':' in item:
-                (key, target) = item.split(':', 1)
+            if '::' in item:
```

Comment:
Add documentation to `parse_target_specs` about what `items` is.

Reply:
In particular, it's confusing that it's quite different than `parse_target` (it was confusing before you made your change, but now that the logic is getting redone, might as well make it clear what it is).